qiskit/aqua/components/optimizers/aqgd.py

```diff
@@ -39,150 +41,307 @@ class AQGD(Optimizer):
 
     Gradients are computed "analytically" using the quantum circuit when evaluating
     the objective function.
-    """
 
-    _OPTIONS = ['maxiter', 'eta', 'tol', 'disp']
+    """
+    _OPTIONS = ['maxiter', 'eta', 'momentum', 'ptol', 'otol', 'averaging']
```

Comment:
ptol and otol do not match to any parameters in the constructor

Reply:
Fixed.